packages/modular-scripts/src/lint.ts

```diff
@@ -54,11 +55,21 @@ async function lint(
     testPathIgnorePatterns: ['/node_modules/', '/dist/'],
   };
 
-  const testArgs = [
-    ...regexes,
-    '--config',
-    `"${JSON.stringify(jestEslintConfig)}"`,
```

Comment:
We previously passed the Jest config via command line with "" around the object, to make it work on windows 
Since updating Jest, it no longer accepts an object with ' " or ` around it. Removing the quotation marks fixed it for Jest on Mac but breaks it for Windows, as it won't accept it as a valid command.

Actually writing the config file and passing the path works, so I've done that as a workaround, generating it in a temp directory, only if running on Windows.